setup.cfg

```diff
@@ -4,3 +4,9 @@ ignore = C901
 
 [pylama:pep8]
 max_line_length = 100
+
+[pylama:pycodestyle]
+max_line_length = 100
+
+[pylama:pylint]
+max_line_length = 100
```

Comment:
I tried that first, but it would cause pycodestyle failures on line_length

With the way I did it, this did not occur...